src/test/java/BaseTest.java

```diff
@@ -1,11 +1,201 @@
+        viewAllBtn.click();
+
+    }
+
+    public void clickFirstSong()
+    {
+        WebElement firstSong = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//section[@id='songResultsWrapper']//td[@class='title']")));
+        firstSong.click();
+    }
+
+    public void clickAddTo()
+    {
+        WebElement addToBtn = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("button[class='btn-add-to']")));
+        addToBtn.click();
+    }
+
+    public void  addToPlayList(String playListName)
+    {
+        //Created Playlist - 'Test1'
+        WebElement playListNameField = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//section[@id='songResultsWrapper']//li[contains(text(),'Test1')]")));
+        playListNameField.click();
+    }
+
+    public String getSuccessAlertMessage()
+    {
+        WebElement successAlertMessage = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//div[@class='success show']")));
+        return successAlertMessage.getText();
+    }
+
+    public void playNextSong()
+    {
+        WebElement playerControlsBtn = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("#mainFooter div[class='side player-controls']")));
+        playerControlsBtn.click();
+        //Thread.sleep(1000);
+        WebElement playNextSongBtn = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("#mainFooter span[class='play']")));
+        playNextSongBtn.click();
+    }
+
+    public boolean isSoundBarVisible()
+    {
+        WebElement soundBar = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("img[alt='Sound bars']")));
+        return soundBar.isDisplayed();
+    }
+
+    public void deleteFromPlaylist(String playList) {
+        //Delete Playlist - 'Test1'
+        WebElement playListNameElement = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//section[@id='playlists']//a[text()='" + playList + "']")));
+        playListNameElement.click();
+        WebElement deletePlaylist= wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("button[class='del btn-delete-playlist']")));
+        deletePlaylist.click();
+
+    }
+
+    public void changePlayListName(String playListName, String newPlayListName)
+    {
+        WebElement playListNameElement = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//section[@id='playlists']//a[text()='" + playListName + "']")));
+        actions.doubleClick(playListNameElement).perform();
+        WebElement playListNameInputField = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("input[name='name']")));
+        playListNameInputField.sendKeys(Keys.chord(Keys.CONTROL,"A",Keys.BACK_SPACE));
+        playListNameInputField.sendKeys(newPlayListName);
+        playListNameInputField.sendKeys(Keys.ENTER);
+
+    } */
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.